tests/test-sha1.js

```diff
@@ -0,0 +1,19 @@
+"use strict";
+
+const test = require("tape");
+const getSha1 = require("../sha1-utils");
+
+
+function isHexString(hashDigest) {
+  for (const character of hashDigest) {
+    if (parseInt(character, 16).toString(16) != character.toLowerCase()) {
+      return false;
+    }
+  }
+  return true;
+}
+
+test("getSha1 returns hex digest", function (t) {
```

Comment:
I notice on L14 of the previous file we were using arrow functions... Not sure if we have a preference or if anything matters. Just thought I'd point it out. I can't imagine there are pros/cons to either approach, apart from consistency.